layouts/index.hbs

```diff
@@ -109,19 +109,17 @@
       <span id="logo-first" class="interactive-item">
         <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
       </span>
-      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive-europe"
-        title="Node.js Interactive Europe">
+      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive-europe">
         <img src="/static/images/interactive/nodejs-interactive-hero-banner-left-eu.png"
-          alt="Amsterdam, Netherlands, September 15-16, 2016"/>
+          alt="Node.js Interactive Europe, Amsterdam, Netherlands, September 15-16, 2016"/>
```

Comment:
I looked at it more closely in dev tools and having the alt tag on the image counted as the content of the anchor, so that overrode the title attribute completely, so the title never would have been read how it was.